lib/store/cart.ts

```diff
@@ -0,0 +1,59 @@
+import {create} from 'zustand';
+import { Product } from '@/app/models/product';
```

Comment:
nie ma sensu tego wynosić gdzieś obok do folderu, mówię o całości `lib` - to wszystko powinno znajdować się w folderze `src` :)